CI failure: k8s is missing runtimeclass resource type

Saw at http://jenkins.katacontainers.io/job/kata-containers-runtime-ubuntu-16-04-PR-initrd/1079/consoleFull
```
# (in test file k8s-pod-quota.bats, line 32)
#   `sudo -E kubectl get deployment "$deployment_name" --output=yaml | grep "forbidden: exceeded quota"' failed
# error: the server doesn't have a resource type "runtimeclass"
# resourcequota/pod-quota created
#     pods: "2"
#     pods: "2"
# deployment.apps/deploymenttest created
# resourcequota "pod-quota" deleted
# deployment.extensions "deploymenttest" deleted
Makefile:69: recipe for target 'kubernetes' failed
make: *** [kubernetes] Error 1
Build step 'Execute shell' marked build as failure
Performing Post build task...
Match found for :.* : True
Logical operation result is TRUE
Running script  : #!/bin/bash
```

--- a/integration/kubernetes/k8s-pod-quota.bats
+++ b/integration/kubernetes/k8s-pod-quota.bats
@@ -18,6 +18,8 @@ setup() {
 @test "Pod quota" {
 	resource_name="pod-quota"
 	deployment_name="deploymenttest"
+	wait_time=10
+	sleep_time=2
 
 	# Create the resourcequota
 	sudo -E kubectl create -f "${pod_config_dir}/resource-quota.yaml"
@@ -29,7 +31,8 @@ setup() {
 	sudo -E kubectl create -f "${pod_config_dir}/pod-quota-deployment.yaml"
 
 	# View information about the deployment
-	sudo -E kubectl get deployment "$deployment_name" --output=yaml | grep "forbidden: exceeded quota"
+	cmd="sudo -E kubectl get deployment \"$deployment_name\" --output=yaml | grep -q 'forbidden: exceeded quota'"
+	waitForProcess "$wait_time" "$sleep_time" "$cmd"
 }
 
 teardown() {

--- a/integration/kubernetes/runtimeclass_workloads/pod-quota-deployment.yaml
+++ b/integration/kubernetes/runtimeclass_workloads/pod-quota-deployment.yaml
@@ -21,3 +21,4 @@ spec:
       containers:
       - name: pod-quota-demo
         image: busybox
+        command: ["tail", "-f", "/dev/null"]

--- a/integration/kubernetes/untrusted_workloads/pod-quota-deployment.yaml
+++ b/integration/kubernetes/untrusted_workloads/pod-quota-deployment.yaml
@@ -23,3 +23,4 @@ spec:
       containers:
       - name: pod-quota-demo
         image: busybox
+        command: ["tail", "-f", "/dev/null"]

